cluster/ci/config/prow/config.yaml

```diff
@@ -269,14 +269,50 @@ presubmits:
     trigger: "((?m)^/test service-catalog,?(\\s+|$))"
 
   openshift/ose:
-  - name: test_pull_requests_ose
+  - name: test_pull_request_ose_check_and_verify_tasks
     agent: jenkins
     labels:
       master: ci.dev.openshift.redhat.com
     always_run: true
     context: ci/openshift-jenkins/ose
-    rerun_command: "/test ose"
-    trigger: "((?m)^/test( all| ose),?(\\s+|$))"
+    rerun_command: "/test check_and_verify_tasks"
+    trigger: "((?m)^/test( all| check_and_verify_tasks),?(\\s+|$))"
+
+  - name: test_pull_request_ose_conformance
+    agent: jenkins
+    labels:
+      master: ci.dev.openshift.redhat.com
+    always_run: true
+    context: ci/openshift-jenkins/ose
+    rerun_command: "/test conformance"
+    trigger: "((?m)^/test( all| conformance),?(\\s+|$))"
+
+  - name: test_pull_request_ose_integration_and_end_to_end
+    agent: jenkins
+    labels:
+      master: ci.dev.openshift.redhat.com
+    always_run: true
+    context: ci/openshift-jenkins/ose
+    rerun_command: "/test networking"
+    trigger: "((?m)^/test( all| integration_and_end_to_end),?(\\s+|$))"
+
+  - name: test_pull_request_ose_networking
+    agent: jenkins
+    labels:
+      master: ci.dev.openshift.redhat.com
+    always_run: true
+    context: ci/openshift-jenkins/ose
+    rerun_command: "/test networking"
+    trigger: "((?m)^/test( all| networking),?(\\s+|$))"
+
+  - name: test_pull_request_ose_verify_commits
```

Comment:
This job should not exist -- fold the single task in it into the check job. This takes like 10s and it doesn't make sense to spin up an entire VM for it